src/index.js

```diff
@@ -1,40 +1,99 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable max-len */
+/* eslint-disable no-alert */
+/* eslint-disable no-unused-expressions */
```

Comment:
[Optional]

- [ ] This is core requirement but since it was missed before it's marked optional, kindly avoid disabling linters in the projects because they're here to impose best practices 💯